spec/features/partner_spec.rb

```diff
@@ -16,6 +16,9 @@
       expect(page.find(:xpath, "//table/tbody/tr[1]/td[1]")).to have_content(@first.name)
       expect(page.find(:xpath, "//table/tbody/tr[3]/td[1]")).to have_content(@third.name)
     end
+    scenario "partner agency stats are visible" do
+      expect(page).to have_text("Partner agencies currently in network")
```

Comment:
*almost* there -- this should also reference the number of partner agencies. The quantity is the important part :)